packages/core/src/lib/oauth/callback.ts

```diff
@@ -142,8 +142,14 @@ export async function handleOAuth(
       throw new Error("TODO: Handle OIDC response body error")
     }
 
-    profile = o.getValidatedIdTokenClaims(result)
+    // profile = o.getValidatedIdTokenClaims(result)
     tokens = result
+
+    if (provider.userinfo?.request) {
```

Comment:
I don't think changes in this file are required at all 🤔 Could you revert?

Reply:
Yeah 😅 I looked at another PR implementing BankID provider, and added after having some trouble. Forgot to remove it :)